enhancements/windows-containers/health-management.md

```diff
@@ -0,0 +1,258 @@
+---
+title: windows-node-health-management
+authors:
+  - "@sebsoto"
+  - "@saifshaikh48"
+reviewers:
+  - "@openshift/openshift-team-windows-containers"
+approvers:
+  - "@aravindhp"
+  - "@sdodson"
+creation-date: 2021-12-16
+last-updated: 2021-12-16
+status: implementable
+---
+
+# Windows Node Health Management
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The purpose of this enhancement is to ensure that all Windows nodes managed by the Windows Machine Config Operator (WMCO)
+maintain a usable state. With health management implemented, WMCO will automatically ensure that OpenShift related services 
+running on the underlying Windows VMs have a status and runtime configuration inline with the expectations of the installed operator version.
+
+## Motivation
```

Comment:
Added motivation is to ease the addition of newer services. I also realize this will make enabling 3rd party networks easier. If we detect a supported 3rd party network, the immutable ConfigMap changes to reflect that i.e. hybrid-overlay service will not be present etc.